### ISSUE TITLE
NPM warning for npmrc entry

### ISSUE THREAD

**Issue by wallzero** [flathub/com.vscodium.codium#161]
When using NPM I receive the following warning:

```
npm WARN config tmp This setting is no longer used.  npm stores temporary files in a special
npm WARN config location in the cache, and they are managed by
npm WARN config     [`cacache`](http://npm.im/cacache).
```

It is possible to remove the `tmp` line from `~/.var/app/com.vscodium.codium/config/npmrc` but not from the Flatpak's read-only `/app/etc/npmrc`.

https://github.com/flathub/com.vscodium.codium/blob/f476f498deb06bdb77faee6ac9f872f1158b6872/npmrc#L6

The warning [does cause some issues](https://github.com/commitizen/cz-cli/issues/964) with command line utilities executed by `npm`.

**Comment by noonsleeper** [comment]
Hey @wallzero, following this [info](https://github.com/flathub/org.freedesktop.Sdk.Extension.node18/issues/27#issuecomment-1397521582), I built a [testing package](https://github.com/flathub/com.vscodium.codium/pull/165#issuecomment-1416539674). Can you try it and say if all is in place without any errors?

**Comment by wallzero** [comment]
Thanks! And sorry I couldn't test it until today. I'm getting a 404 when I try to pull the flatpak. Is it possibly expired?

**Comment by noonsleeper** [comment]
Yes indeed, the build's lifetime is 24hs. I'll run a new [build](https://github.com/flathub/com.vscodium.codium/pull/165#issuecomment-1424556821) for you.

**Comment by wallzero** [comment]
I tested the build today and didn't run into any issues. The error is indeed gone and it no longer causes issues with some terminal command interfaces.
